pkg/kernelrelease/kernelrelease.go

```diff
@@ -30,13 +31,32 @@ var SupportedArchs = Architectures{
 // Privately cached at startup for quicker access
 var supportedArchsSlice []string
 
+// Represents the minimum kernel version for which building the module
+// is supported, depending on the architecture
+var moduleMinKernelVersion map[Architecture]semver.Version
+
+// Represents the minimum kernel version for which building the probe
+// is supported, depending on the architecture
+var probeMinKernelVersion map[Architecture]semver.Version
+
 func init() {
 	i := 0
 	supportedArchsSlice = make([]string, len(SupportedArchs))
 	for k := range SupportedArchs {
 		supportedArchsSlice[i] = k.String()
 		i++
 	}
+
+	// see compatibility matrix: https://falco.org/docs/event-sources/drivers/
+	// note: this does not make much sense for flatcar, which has a much
+	// higher major version. In that case, we assume that the module/probe
+	// is always supported, and eventually fail while building
+	moduleMinKernelVersion = make(map[Architecture]semver.Version)
```

Comment:
Can we statically initialize this as a global, like we do for `var images` in builders.go?
I find it more readable and "fixed", to be also used as a documentation.